src/batch-client/index.ts

```diff
@@ -433,6 +436,57 @@ export class ZimbraBatchClient {
 			return ids ? ids.split(',') : [];
 		});
 
+	// For offline Drafts
+	public attach = (files: any, { ...message }) => {
+		const promises = castArray(files).map(file => this.localStoreClient.attach({ file }));
+
+		return Promise.all(promises).then(attached => {
```

Comment:
can we .catch()

Reply:
Please review https://github.com/ZimbraOS/zm-x-web/pull/3473